09_structure_filtering/src/main/java/pl/samouczekprogramisty/szs/filtering/MyStructure.java

```diff
@@ -8,6 +9,10 @@
 
 public class MyStructure implements IMyStructure {
     private final List<INode> nodes = new LinkedList<>();
+    
+    public MyStructure(INode... nodes) {
+		Arrays.asList(nodes).stream().forEach(node -> this.nodes.add(node));
```

Comment:
Co myślisz o takiej zmianie?
```suggestion
		Arrays.asList(nodes).stream().forEach(this::addNode);
```